Source/controls/plrctrls.cpp

```diff
@@ -410,7 +409,7 @@ void FindTrigger()
 			if (pcursmissile != -1 && distance == newDistance && rotations < newRotations)
 				continue;
 			cursPosition = missile.position.tile;
-			pcursmissile = mi;
+			pcursmissile = missileWithId.first;

```

Comment:
`pcursmissile` could be a pointer or an iterator into the container to save needing to reference the key here (and remove the need to do a lookup when the value is actually used).

Reply:
I changed it to a pointer. 😉